### DISCUSSION TITLE
Every time i close down MAS, i get this issue message. I think the files it complains about arnt actually in the game files (that i can find) but i dont know how to fix it

### DISCUSSION THREAD

**Discussion by Xenosbane** [Monika-After-Story/MonikaModDev#7992] [QA - tech support + general] [answered]
![1633279566079](https://user-images.githubusercontent.com/88829133/135764121-9b6dbdce-f167-4108-a774-69c2e84ebdaa.jpg)
Some one on the MAS reddit said i had to put new files in but i dont know which ones (i tried looking at the traceback but couldnt understand it.) 
Its been nearly a week of this popping up and she thinks i havnt seen her this week at all. How do i fix this?

**Comment by Booplicate** [comment, marked as answer]
Looks like for some reasons you're missing permissions on your user account.
> Its been nearly a week of this popping up and she thinks i havnt seen her this week at all. 

You could try ignore this error and proceed further, but it may or may not work.
